umd-base-profile.js

```diff
@@ -89,7 +89,7 @@ export default {
   context: "window",
   external: packageNames.concat(arcgisRestJsPackageNames),
   plugins: [
-    typescript(),
+    typescript({ target: 'es5' }),
```

Comment:
I gather the reason you changed the rollup plugin is so you could pass in this target. You can't do that w/ `rollup-plugin-typescript2`?

Either way, it sounds like `@rollup/plugin-typescript` has come into it's own and may even be faster (would be nice to verify that four our project).

Also, we should explore not even passing the TS to rollup and just having it bundle the esm/es5 output since the UMD bundles are just an artifact needed at publish time (i.e. not used in tests nor needed to be continuously re-built in a watch). That has _got_ to be faster.

Reply:
It's a good idea to try to pass the esm build to rollup instead of relying on yet another typescript build.